theforeman.org/scripts/debian/stage_packages.sh

```diff
@@ -1,25 +1,57 @@
 #!/bin/bash
-set -xe
+set -xeu
 
 # Script to copy the newly-built debs to the web node for signing and promoting
 # Dependencies
 # * the freight::client class in foreman-infra
 # * the web node must have the freight class applied
 
 # Find the deps in the build-dir from the previous step
-DEB_PATH=./debian/${os}/build-${project}
+for possible_path in ./debian/${os}/build-${project} ./plugins/build-${project} ./dependencies/${os}/build-${project}; do
+  if [[ -d "${possible_path}" ]]; then
+    DEB_PATH="${possible_path}"
+  fi
+done
+
+if [[ -z $DEB_PATH ]] ; then
```

Comment:
the script is now `-u` so this will explode and the `echo` won't trigger.
```suggestion
if [[ -z ${DEB_PATH:-} ]] ; then
```